### PR TITLE
Fix Qunit tests for latest Google Chrome (close #217)

### LINKED ISSUE #217: Fix Qunit tests for latest Google Chrome (47.0.2526.73)

### PR DESCRIPTION
**Pull request by helen-dikareva**
/cc @inikulin @AndreyBelym

### CODE REVIEW

File: src/client/runner/automation/playback/type.js [hunk 1]
```diff
@@ -152,21 +152,26 @@ export default function (el, text, options, actionCallback) {
...
-                                            ((isInputWithoutSelectionProperties && /(\.|-)/.test(prevChar)) ||
-                                             (!browserUtils.isIE9 && $(elementForTyping).is('input[type=number]') &&
-                                              prevChar === '-')))
-                                            typeCharPlaybackAutomation(elementForTyping, prevChar +
-                                                                                         currentChar, caretPosition +
-                                                                                                      currPos - 1);
-                                        else
-                                            typeCharPlaybackAutomation(elementForTyping, currentChar, caretPosition +
-                                                                                                      currPos);
+                                    var currentChar       = text.charAt(currPos),
```
Comment (inikulin) [inline src/client/runner/automation/playback/type.js]: Var declaration style is incorrect

File: src/client/runner/automation/playback/type.js [hunk 2]
```diff
@@ -152,21 +152,26 @@ export default function (el, text, options, actionCallback) {
...
+                                        prevChar          = currPos === 0 ? null : text.charAt(currPos - 1),
+                                        isInputTypeNumber = elementForTyping.tagName.toLowerCase() === 'input' &&
+                                                            elementForTyping.type === 'number',
+                                        problemBrowsers   = (browserUtils.isFirefox ||
```
Comment (inikulin) [inline src/client/runner/automation/playback/type.js]: It's `problematicBrowsers`

**Comment by inikulin** [comment]
\r-

**Comment by inikulin** [comment]
FPR?

**Comment by helen-dikareva** [comment]
FPR

**Comment by inikulin** [comment]
lgtm

**Comment by AndreyBelym** [comment]
lgtm

**Pull request merged**